sanic/config.py

```diff
@@ -1,10 +1,75 @@
+from sanic.defaultFilter import DefaultFilter
 import os
-
 import types
 
 SANIC_PREFIX = 'SANIC_'
 
 
+DEFAULT_LOG_CONF = {
+    'version': 1,
+    'filters': {
+        'access_filter': {
+            '()': DefaultFilter,
+            'param': [0, 10, 20]
+        },
+        'error_filter': {
+            '()': DefaultFilter,
+            'param': [30, 40, 50]
+        }
+    },
+    'formatters': {
+        'simple': {
+            'format': '%(asctime)s - (%(name)s)[%(levelname)s]: %(message)s',
+            'datefmt': '%Y-%m-%d %H:%M:%S'
+        },
+        'access': {
+            'format': '%(asctime)s - [%(levelname)s][%(host)s]: ' +
+                      '%(request)s %(message)s %(status)d %(byte)d',
+            'datefmt': '%Y-%m-%d %H:%M:%S'
+        }
+    },
+    'handlers': {
+        'internal': {
+            'class': 'logging.handlers.TimedRotatingFileHandler',
```

Comment:
I think it should log to `sys.stderr` by default. Set `filename` to `access.log` won't help much because most people don't want to write logs to current working directory, they most likely want to write logs to `/var/log/xxx/access.log` or simply log to `syslog`.